polaris-react/src/components/ActionList/components/Item/Item.tsx

```diff
@@ -88,7 +88,9 @@ export function Item({
       </Text>
     </>
   ) : (
-    contentText
+    <Text as="span" variant="bodyMd">
```

Comment:
```suggestion
    <Text as="span" variant="bodyMd" fontWeight={active ? "semibold" : "regular"}>
```

In the storybook diff for [ActionList with an icon and suffix](https://5d559397bae39100201eedc1-yrjhwxlxrv.chromatic.com/?path=/story/all-components-actionlist--with-an-icon-and-a-suffix), the active item no longer has the bolded font weight ([prod](https://storybook.polaris.shopify.com/?path=/story/all-components-actionlist--with-an-icon-and-a-suffix)).

Reply:
Great catch!